From 0562c5b92378610dacd83ccfbd66b32dbcdb56c7 Mon Sep 17 00:00:00 2001
From: Amber Lin <Amber.Lin@amd.com>
Date: Wed, 24 May 2017 10:39:57 -0400
Subject: [PATCH 2214/2500] Revert "drm/amdgpu: Disable SDMA-ULV interrupts"

This reverts 'commit a035ff50ae69 ("drm/amdgpu: Disable SDMA-ULV
interrupts")'. SDMA firmware fixes the issue. This workaround is no
longer needed.

Change-Id: I6d9596ff05bbd3e1acf235e4b097a793183cda85
Signed-off-by: Amber Lin <Amber.Lin@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/sdma_v4_0.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/sdma_v4_0.c b/drivers/gpu/drm/amd/amdgpu/sdma_v4_0.c
index d1c45da..121290a 100644
--- a/drivers/gpu/drm/amd/amdgpu/sdma_v4_0.c
+++ b/drivers/gpu/drm/amd/amdgpu/sdma_v4_0.c
@@ -687,13 +687,6 @@ static int sdma_v4_0_gfx_resume(struct amdgpu_device *adev)
 
 		if (adev->mman.buffer_funcs_ring == ring)
 			amdgpu_ttm_set_active_vram_size(adev, adev->mc.real_vram_size);
-
-		/* FIXME: temporarily disable SDMA-ULV interrupts for Vega10.
-		 * Remove this once the fix is in firmware.
-		 */
-		if (ring->adev->asic_type == CHIP_VEGA10)
-			WREG32(sdma_v4_0_get_reg_offset(i, mmSDMA0_ULV_CNTL),
-							0);
 	}
 
 	return 0;
-- 
2.7.4

